.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,32 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
```

Comment:
remove this one. check-standard is sufficient.

Reply:
Done